backend/pokemons/models.py

```diff
@@ -7,6 +7,7 @@ class Pokemon(models.Model):
     attack = models.IntegerField()
     defense = models.IntegerField()
     hit_points = models.IntegerField()
+    image = models.ImageField(blank=True, max_length=100)
```

Comment:
No need to set the `max_length` here, it's 100 by default

Reply:
You should set the `upload_to` param. Go read about it in `FileField` docs: https://docs.djangoproject.com/en/3.1/ref/models/fields/#django.db.models.FileField

Reply:
Thanks for the references!